api/stats/events.proto

```diff
@@ -46,6 +47,11 @@ message Hit {
 
     optional bool clear = 23;
 
+    // Scales from 100 to 0;
+    // 100 indicates an opponent was right on top of this hit.
+    // 0 indicates the hit was made with little to no defending pressure.
+    optional int32 pressure = 24;
```

Comment:
Should this exist in 50s?

Reply:
Ah, this is supposed to have a diff base with the hit-pressure branch. Not sure how to indicate that on git. Can fix.